src/icons.h

```diff
@@ -252,6 +253,7 @@ static const struct icon_pair icons_ext[] = { /* All entries are case-insensitiv
 	{"deb",        ICON_EXT_DEB,        COLOR_ARCHIVE},
 	{"diff",       ICON_EXT_DIFF,       0},
 	{"dll",        ICON_SCRIPT,         0},
+	{"djvu",       ICON_DJVU,           0},
```

Comment:
Should use the same color as pdf, epub etc.

```suggestion
	{"djvu",       ICON_DJVU,           COLOR_DOCS},
```